processor/src/main/java/org/mapstruct/ap/model/MappingMethod.java

```diff
@@ -49,11 +50,11 @@
     private final String resultName;
 
     /**
-     * constructor to be overloaded when local variable names are required prior to calling this constructor. (e.g.
-     * for property mappings). It is supposed to be initialized with at least the parameter names.
+     * constructor to be overloaded when local variable names are required prior to calling this constructor. (e.g. for
+     * property mappings). It is supposed to be initialized with at least the parameter names.
      *
-     * @param method
-     * @param existingVariableNames
+     * @param method method
```

Comment:
Personally I would have ommitted the param tags altogether. These shouldn't cause any errors / build failures btw. The reporting of JavaDoc tooling is really bad; it fails the build as soon as there actually is one real "error" (such as broken links or actual syntax errors), but these warnings also show up. It's a bit cumbersome to identify the actual trouble makers.

Same for the return tags. They should be fine in the original form.